src/resources/enterpriseSettings.js

```diff
@@ -0,0 +1,5 @@
+import createResourceDescription from './util/createResourceDescription'
+
+export const enterpriseSettings = createResourceDescription(
+  'enterprise_settings'
```

Comment:
According to the README the enterprise_settings should be fetched like this:
```
SDK.enterprise()
  .settings()
  .then(response => response.data);
```

currently it's:
```
SDK.enterpriseSettings().then(...)
```

Reply:
Yeah this is because it is a diversion from the real api, maybe we should put it in shorthands?
Or we should add the ability to add shorthands onto resources (and sub resources) as well. So that we can define them inside the resource definition?

Reply:
maybe we need to discuss with @christophethelen ?

Reply:
If there is a real advantage on that (like faster development), then we can do that. Otherwise I would only do that when you feel that it starts to hinder development

Reply:
Adding SDK.enterprise().settings() is a bit slower for now because it requires additional development. And it would be a bit confusing because it does not produce the url you expect. You would expect /enterprise/id/settings but now you get /enterprise_settings instead. I personally would keep it like this: `SDK.enterpriseSettings().then(...)` for now.